prog/vm/github_runner.rb

```diff
@@ -165,9 +165,16 @@ def before_run
     hop_setup_environment
   end
 
-  def concurrency_available?
+  def quota_available?
     github_runner.installation.project_dataset.for_update.all
-    github_runner.installation.project.runner_core_limit > github_runner.installation.project.github_installations.sum(&:total_active_runner_cores)
+    # In existing Github quota calculations, we compare total allocated cores
+    # with the core limit and allow passing the limit once. This is because we
+    # check quota and allocate VMs in different labels hence transactions and it
+    # is difficult to enforce quotas ih the environment with lots of concurrent
+    # requests. There are some remedies, but it would require some refactoring
+    # that I'm not keen to do at the moment. Although it looks weird, passing 0
+    # as requested_additional_usage keeps the existing behavior.
+    github_runner.installation.project.quota_available?("GithubRunnerCores", 0)
```

Comment:
If I merge my pull request before yours. https://github.com/ubicloud/ubicloud/pull/1782

```suggestion
    project.quota_available?("GithubRunnerCores", 0)
```